stdlib/public/Concurrency/CheckedContinuation.swift

```diff
@@ -279,8 +281,8 @@ public func withCheckedContinuation<T>(
 ///   - body: A closure that takes an `UnsafeContinuation` parameter.
 ///     The closure must resume the continuation exactly once.
 ///
-/// If the closure throws an error,
-/// this function rethrows that error.
+/// If the closure calls `resume(throwing:)` on the continuation,
```

Comment:
The closure isn't the only thing that can use the continuation; it's allowed to store it anywhere for use later, so even though it's passive voice, it might be more accurate to say:

> If `resume(throwing:)` is called on the continuation, this method throws that error.

Reply:
Changed in d101b39b7a4.